docs/ABOUT.md

```diff
@@ -1,7 +1,47 @@
+Python is a powerful programming language, it is well known as a "glue" that means: it is used with many other technologies. It is multi platform, runs everywhere. Is friendly and easy to learn. It is licensed as open source, most versions are compatible with the GPL with a few exceptions on old versions. But, one of the most cool things about Python is the community that is very helpful and collaborative. Take a look at: [About python](https://www.python.org/about/).
+
 Python is a strong language for beginners. There are many resources available for programmers of all levels, the code is highly readable, and in many cases phrases are comparable to those in the English language. Code can be written and executed from the command line, in an interactive IPython session, or in a [Jupyter](http://jupyter.org) (IPython) notebook.
 
 The most common form of Python is compiled in C. This is often invisible to the beginning programmer, but if there are uses for which exceptionally fast implementation is needed then C extensions can be written to optimize Python execution.
 
 [Python is used extensively](https://www.python.org/about/apps/) in scientific computing, finance, games, networking, internet development, and in assembling pipelines of other programs.
 
 Python was started by Guido van Rossum in 1989. Its name is an homage to the comedy troupe Monty Python. Python 2 is used widely but support may end by 2020. Python 3 was introduced in 2008 and is beginning to be adopted more widely. They are similar, but users will encounter [some differences](http://blog.teamtreehouse.com/python-2-vs-python-3). Python development is shepherded by [The Python Software Foundation](https://www.python.org/about/) and there are active community-based user groups worldwide.
+
+### What python is good for?
+
+Python can be applied to many different classes of problems [what python is good for](https://docs.python.org/3/faq/general.html#what-is-python-good-for). It have a powerful standard library with string processing, Internet protocols, software engineering, operating system interfaces. Python also have a lot of third-party modules, easily found in The Python Package Index (PyPi). Python software covers areas such as:
+
+- [Web and Internet Development](https://www.python.org/about/apps/#web-and-internet-development)
+- [Database Access](https://www.python.org/about/apps/#database-access)
+- [Desktop GUIs](https://www.python.org/about/apps/#desktop-guis)
+- [Scientific & Numeric](https://www.python.org/about/apps/#scientific-and-numeric)
+- [Education](https://www.python.org/about/apps/#education)
+- [Network Programming](https://www.python.org/about/apps/#network-programming)
+- [Software & Game Development](https://www.python.org/about/apps/#software-development)
+
+### Who uses python?
```

Comment:
I don't think we need this section—we're already linking to the about page, and if people want to know more they'll find it in that about link.